src/accelerate/components/List.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
 import { useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
+import { Pagination } from "react-pagination-bar"
```

Comment:
```suggestion
import { Pagination } from 'react-pagination-bar';
```
Single quotes for imports and always use a semi-colon. If in doubt look at the existing code style in the file and keep it consistent with that.